securedrop/tests/functional/test_admin_interface.py

```diff
@@ -1,53 +1,476 @@
-from tbselenium.utils import SECURITY_LOW
+import logging
+import time
+from pathlib import Path
+from typing import Callable, Generator, Tuple
+from uuid import uuid4
 
-from . import functional_test as ft
-from . import journalist_navigation_steps, source_navigation_steps
+import pytest
+from models import Journalist
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.firefox.webdriver import WebDriver
+from selenium.webdriver.support import expected_conditions
+from tests.functional import functional_test as ft
+from tests.functional import journalist_navigation_steps, source_navigation_steps
+from tests.functional.app_navigators import JournalistAppNavigator
+from tests.functional.conftest import SdServersFixtureResult, spawn_sd_servers
+from tests.functional.db_session import get_database_session
+from tests.functional.factories import SecureDropConfigFactory
+from tests.functional.sd_config_v2 import SecureDropConfig
 
 
-class TestAdminInterface(
```

Comment:
I split this class into `TestAdminInterfaceAddUser`, `TestAdminInterfaceEditAndDeleteUser` and `TestAdminInterfaceEditConfig`.